File: c프로그래밍 실습/4주차/1.c
#include<stdio.h>
#include <math.h>

int main(void) {
	// 변수 선언
	int input, i;
	int goal = 50;
	int min = 0;
	int max = 100;
	//------------------------------------------
	for (i = 1;; i++) {
		// 입력받기
		printf("How about %d? (try more : 1, try less : 2, right : 0) ", goal);
		scanf("%d", &input);
		
		if (input == 1) min = goal; // 최솟값 재설정
		else if (input == 2) max = goal; // 최댓값 재설정
		else if (input == 0) break; // 종료
		goal = (min + max) / 2; // 중간값으로 초기화
		if (((float)min + (float)max) / 2.0 - goal >= 0.5) goal++; // 반올림
	}
	printf("%d tried.", i); // 결과 출력

	return 0;
}

File: 파이썬 프로그래밍/퀴즈1/1.py
# 입력
value = int(input("수를 입력하세요 : "))

if value == 0:  # 변수가 0 이라면
    print("0입니다")
elif value % 3 == 0:  # 변수가 3의 배수라면
    if value % 2 == 1:  # 홀수
        print("3의 배수인 홀수입니다")
    else:
        print("3의 배수인 짝수 입니다")
else:  # 3의 배수가 아니라면
    if value % 2 == 1:  # 홀수
        print("홀수입니다")
    else:
        print("짝수 입니다")
